bridges/drm_bridge_if.m

```diff
@@ -40,3 +41,12 @@
 	struct drm_encoder	*encoder;
 	struct drm_device	*drm;
 };
+
+#
+# Add the encoder to the drm pipeline
+#
+METHOD int add_encoder {
```

Comment:
That doesn't seems correct to put this here.
I guess this is because you need a dummy encoder between your crtc and this bridge ?
Can you share more about the design of the Morello board on this ?

Reply:
Morello board has NXP HDMI controller off the SoC, so I thought we want some abstraction here to ensure it works on another boards based on morello SoC (other boards may have another bridge).

Reply:
Ok but adding an encoder in the bridge interface doesn't seems logical to me.
What does the pipeline looks like for the Morello board ?

Reply:
komeda is a single memory-mapped device (in terms of DTS nodes), it consists of multiple components:
1) layer processing unit, here we can configure up to 4 layers
2) composition unit. this device merges layers and feeds DOU
3) display output unit (DOU), this device passes stream to HDMI controller